identify/extensions.py

```diff
@@ -4,62 +4,138 @@
 
 
 EXTENSIONS = {
+    'apinotes': {'text', 'apinotes'},
+    'asar': {'binary', 'asar'},
+    'bash': {'text', 'shell', 'bash'},
     'bat': {'text', 'batch'},
     'bz2': {'binary', 'bzip2'},
     'c': {'text', 'c'},
     'cc': {'text', 'c++'},
+    'cfg': {'text', 'conf'},
```

Comment:
yeah, not sure what to do. obviously `.conf`/`.cfg`/`.cnf` represent tons of different file formats

maybe we should just omit it (or just have `{'text'}`) since you can't reliably assume anything based on the type?